net/pfSense-pkg-cellular/files/usr/local/sbin/cellular.py

```diff
@@ -34,524 +34,527 @@
 import ConfigParser
 import serial
 
-__version__ = "1.1.0_B22"
+__version__ = "1.1.6"
 
 huawei = (lambda a: a.startswith("Huawei"))
 
 class CellularInterface:
-	"""
```

Comment:
Is there a compelling reason for changing this entire file from tabs to spaces? It makes the changes difficult to find and review.

Reply:
Hi,
no sorry I just fixed up the indentation which was a little all over the place.
The line that you marked was the only one that changed/was added in this file.